src/deploy/functions/release.js

```diff
@@ -367,7 +367,13 @@ module.exports = function(context, options, payload) {
         .filter({ state: "fulfilled" })
         .map("value")
         .value();
-      failedDeployments += failedCalls.length;
+      failedDeployments = failedCalls.map((error) => {
+        if (error.context && error.context.function) {
```

Comment:
`_.get(error, "context.function", "")` -> gets the path if it exists, or the default value (third parameter)